internal/component/azure/eventhubs/eventhubs.go

```diff
@@ -138,11 +152,94 @@ func (aeh *AzureEventHubs) Publish(ctx context.Context, topic string, messages [
 	return nil
 }
 
+// GetPubSubHandlerFunc returns the handler function for pubsub messages
+func (aeh *AzureEventHubs) GetBindingsHandlerFunc(topic string, getAllProperties bool, handler bindings.Handler, timeout time.Duration) HandlerFn {
+	return func(ctx context.Context, messages []*azeventhubs.ReceivedEventData) ([]HandlerResponseItem, error) {
+		if len(messages) != 1 {
```

Comment:
Trying to understand: Why is it expecting only 1 message here?

Reply:
It's expecting one message only because Bindings doesn't have bulk handler right? So only one message can be sent in one invocation of handler.